### DISCUSSION TITLE
React Native Google Auth Strapi.

### DISCUSSION THREAD

**Discussion by shtaimoortariq** [strapi/strapi#8775] [Legacy Topics]
does anyone know how to Login via ReactNative App with Strapi Google Auth Providers i have a Webapp on react and it is working like a charm but now i am trying to implement it in React-native how do i integrate does anybody know ??

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question, we are currently planning to decomission our GitHub Discussions in early 2021 and we don't want to lose the great questions asked by our community.

Questions should be directed to [our forum](https://forum.strapi.io/). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).
